doc/api/deprecations.md

```diff
@@ -2321,7 +2321,6 @@ Type: Runtime
 
 Please use `Server.prototype.setSecureContext()` instead.
 
-
```

Comment:
Unrelated whitespace change?

Reply:
We use single line breaks throughout the whole file besides here. That's why I removed it. Should I put it back in?

Reply:
It's fine by me.